src/main/java/kr/syeyoung/dungeonsguide/YoMamaOutdated.java

```diff
@@ -0,0 +1,116 @@
+package kr.syeyoung.dungeonsguide;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.*;
+import net.minecraftforge.client.event.GuiOpenEvent;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.fml.common.FMLCommonHandler;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import org.apache.commons.io.IOUtils;
+import org.apache.http.Header;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.message.BasicHeader;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.lwjgl.opengl.GL11;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public class YoMamaOutdated {
+
+    Logger logger = LogManager.getLogger("YoMamaOutdated");
+
+    public boolean isUsingOutdatedDg = true;
+    String outdatedMessage;
+
+    public YoMamaOutdated() {
+        MinecraftForge.EVENT_BUS.register(this);
+        this.check();
+    }
+
+    void check() {
+
+        try (CloseableHttpClient httpclient = HttpClients.createDefault()) {
+            HttpGet httpget = new HttpGet( "https://dungeonsguide.kingstefan26.workers.dev/outdated");
```

Comment:
ehhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh and h goes on forever